### PR TITLE
Management command to download OSM extracts for relation ids

### PR DESCRIPTION
**Pull request by smn**
This accompanies the `import_geojson` management command. It takes OSM relation ids as positional arguments and then downloads the relevant files from a GitHub repository. It defaults to `nyaruka/posm-extracts`.

### CODE REVIEW

File: temba/locations/management/commands/download_geojson.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+class Command(BaseCommand):
+    help = 'Download geojson files for OSM relation ids.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('relation_ids', nargs='+')
+        parser.add_argument('--oauth-token',
+                            dest='oauth_token',
+                            default=None,
+                            help='The OAuth token to use when authenticating '
```
Comment (rowanseymour) [inline temba/locations/management/commands/download_geojson.py]: max-line-length=120 ...

File: temba/locations/management/commands/download_geojson.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import os
```
Comment (rowanseymour) [inline temba/locations/management/commands/download_geojson.py]: `from __future__ import print_function, unicode_literals`

File: temba/locations/management/commands/download_geojson.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import requests
+import re
```
Comment (rowanseymour) [inline temba/locations/management/commands/download_geojson.py]: We've been switching to `regex` instead of `re`

**Comment by smn** [comment]
@rowanseymour thanks, updated.

The reason I added this was so that we could automate the geojson stuff using environment variables in a docker deploy. If not of interest here I can ship it as part of the docker builds.

**Comment by smn** [comment]
@nicpottier 👍  👎  ?

**Comment by nicpottier** [comment]
Sorry, tried to comment on this over email but apparently didn't work.

This is definitely super cool, note that the Nyaruka extracts are definitely stale but this is a nice bootstrap. Someday when we have lots of spare time I really want to build these automatically and publish them on a fancy site, but ya, not a lot of that.

**Pull request merged**